Last-Update: 2012-10-04
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Bug-Zabbix: https://support.zabbix.com/browse/ZBX-4986
Description: fix columns sorting in frontend-php
 Problem is manifesting with nginx/php-fpm
 or with Apache/mod_proxy.

--- a/frontends/php/include/func.inc.php
+++ b/frontends/php/include/func.inc.php
@@ -1529,9 +1529,9 @@
 	if ($page['type'] != PAGE_TYPE_HTML && defined('ZBX_PAGE_MAIN_HAT')) {
 		$script = "javascript: return updater.onetime_update('".ZBX_PAGE_MAIN_HAT."', '".$url."');";
 	}
 	else {
-		$script = 'javascript: redirect("'.$url.'");';
+		$script = 'javascript: redirect("'.basename($url).'");';
 	}
 
 	zbx_value2array($obj);
 	$cont = new CSpan();
